### PR TITLE
add post operation

### PR DESCRIPTION
**Pull request by jesuspg**
@jframos

### CODE REVIEW

File: lib/routes/v1.js [hunk 1]
```diff
@@ -75,20 +75,71 @@ function getKey(req, res) {
...
+
+        var filename = keysPath + region + '.' + key + 'key';
+
+        fs.writeFile(filename, req.body, function(err) {
+            if (err) {
+                return console.log(err);
+            }
+
+            console.log('The key file %s was saved!', filename);
+            res.status(201);
```
Comment (jframos) [inline lib/routes/v1.js]: Perhaps we might return the created key in the response.

**Comment by jframos** [comment]
I think it is important to check the **Content-Type** header in the case of POST requests. We only should accept **text/plain** values and return a *HTTP 415 Unsupported Media Type* if the Content-Type is not the expected one.

File: lib/routes/v1.js (hunk 1 above)
Comment (jesuspg) [inline lib/routes/v1.js]: done!

**Comment by jframos** [comment]
LGTM

**Pull request merged**